cgsmiles/graph_utils.py

```diff
@@ -114,3 +114,17 @@ def annotate_fragments(meta_graph, molecule):
         meta_graph.nodes[meta_node]['graph'] = graph_frag
 
     return meta_graph
+
+
+def set_atom_names_atomsitic(meta_graph, molecule):
+    """
+    Set atomnames according to commonly used convention
+    in molecular dynamics (MD) forcefields. This convention
+    is defined as element plus counter for atom in residue.
+    """
+    for meta_node in meta_graph.nodes:
+        fraggraph = meta_graph.nodes[meta_node]['graph']
+        for idx, node in enumerate(fraggraph.nodes):
+            atomname = fraggraph.nodes[node]['element'] + str(idx)
```

Comment:
Should those numbers be 0-padded (e.g. `H01` rather than `H1`)? I'd also prefer a formatting string here.

Reply:
Personally, I'm not a fan of zero padding. It makes handling them unnecessarily more complicated. There is no agreement on that part anyways. So it is more like pirate rules rather than convention